pythonforandroid/recipes/zbar/__init__.py

```diff
@@ -4,14 +4,11 @@
 
 class ZBarRecipe(PythonRecipe):
 
-    version = '0.10'
+    version = '0.23.92'
```

Comment:
Version `0.23.92` does not exists.

Are you referring to another recipe?

Reply:
I think this is maintained fork https://github.com/mchehab/zbar/